test/server/paymentProviders/transferwise/index.test.ts

```diff
@@ -0,0 +1,138 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import * as utils from '../../../utils';
+import { fakeCollective, fakeConnectedAccount, fakeExpense } from '../../../test-helpers/fake-data';
+import transferwise from '../../../../server/paymentProviders/transferwise';
+import * as transferwiseLib from '../../../../server/lib/transferwise';
+
+const quote = {
+  source: 'USD',
+  target: 'EUR',
+  sourceAmount: 101.14,
+  targetAmount: 90.44,
+  rate: 0.9044,
+  fee: 1.14,
+};
+const createQuote = sinon.stub(transferwiseLib, 'createQuote').resolves(quote);
+sinon.stub(transferwiseLib, 'getTemporaryQuote').resolves(quote);
+sinon.stub(transferwiseLib, 'getProfiles').resolves([
+  {
+    id: 217896,
+    type: 'personal',
+  },
+  {
+    id: 220192,
+    type: 'business',
+  },
+]);
+const createRecipientAccount = sinon.stub(transferwiseLib, 'createRecipientAccount').resolves({
+  id: 13804569,
+  accountHolderName: 'Leo Kewitz',
+  currency: 'EUR',
+  country: 'DE',
+  type: 'iban',
+  details: {
+    IBAN: 'DE89370400440532013000',
+  },
+});
+const createTransfer = sinon.stub(transferwiseLib, 'createTransfer').resolves({ id: 123 });
+const fundTransfer = sinon.stub(transferwiseLib, 'fundTransfer').resolves({ status: 'COMPLETED' });
+
+describe('paymentMethods.transferwise', () => {
+  let connectedAccount, host, payee, expense;
+
+  after(sinon.restore);
+  before(utils.resetTestDB);
+  before(async () => {
+    host = await fakeCollective({ isHostAccount: true });
+    connectedAccount = await fakeConnectedAccount({
+      CollectiveId: host.id,
+      service: 'transferwise',
+      token: '33b5e94d-9815-4ebc-b970-3612b6aec332',
+      data: { type: 'business', id: 0 },
+    });
+    payee = await fakeCollective({
+      settings: { hasBankAccount: true },
+      data: {
+        bankAccount: {
+          accountHolderName: 'Leo Kewitz',
+          currency: 'EUR',
+          type: 'iban',
+          legalType: 'PRIVATE',
+          details: {
+            IBAN: 'DE89370400440532013000',
+          },
+        },
+      },
+    });
+    expense = await fakeExpense({
+      payoutMethod: 'transferwise',
```

Comment:
This field is not going to be there with the payout method refactor. I'd recommend waiting for https://github.com/opencollective/opencollective-api/pull/3231 to be merged before going too far, otherwise you're gonna face multiple conflicts and design issues.